packages/xrpl/test/integration/transactions/escrowCreate.test.ts

```diff
@@ -0,0 +1,65 @@
+import { assert } from 'chai'
+import { EscrowCreate } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { generateFundedWallet, testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('EscrowCreate', () => {
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'base',
+    async () => {
+      const wallet1 = await generateFundedWallet(testContext.client)
+
+      // get the most recent close_time from the standalone container for finish after.
+      const CLOSE_TIME: number = (
+        await testContext.client.request({
+          command: 'ledger',
+          ledger_index: 'validated',
+        })
+      ).result.ledger.close_time
+
+      const finishAfterPromise = new Promise<void>((resolve) => {
+        setTimeout(resolve, 3000)
+      })
+
+      const tx: EscrowCreate = {
+        Account: testContext.wallet.classicAddress,
+        TransactionType: 'EscrowCreate',
+        Amount: '10000',
+        Destination: wallet1.classicAddress,
+        FinishAfter: CLOSE_TIME + 2,
+      }
+
+      await testTransaction(testContext.client, tx, testContext.wallet)
+
+      await finishAfterPromise
```

Comment:
What does the `finishAfterPromise` change?

Reply:
Seems like it isn't needed here, removing